staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -1348,6 +1348,10 @@ type PodNetwork struct {
 	// CIDR for vm network.
 	// Default 10.0.2.0/24 if not specified.
 	VMNetworkCIDR string `json:"vmNetworkCIDR,omitempty"`
+
+	// IPv6 CIDR for the vm network.
+	// Defaults to fd10:0:2::/120 if not specified.
+	VMIpv6NetworkCIDR string `json:"vmIpv6NetworkCIDR,omitempty"`
```

Comment:
nit: `json:vmIPv6NetworkCIDR` should be the more correct field name.

Reply:
I agree, but this is following the IPv4 format and I am not sure if it is better to break the consistency here.

Reply:
I could not find an `Ipv4` field in the schema, could you point me to the field which you mean?

Reply:
Ahh... I am wrong here. I thought about something else completely. Sorry.
Changing...

Reply:
Done

Reply:
:+1: